integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -12,18 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pyspark.sql import SparkSession
+try:
+    from pyspark.sql import SparkSession
+except ImportError as error:
+    import findspark
+    findspark.init()
+    from pyspark.sql import SparkSession
 
 def _spark__init():
     #Force the RapidsPlugin to be enabled, so it blows up if the classpath is not set properly
     # DO NOT SET ANY OTHER CONFIGS HERE!!!
     # due to bugs in pyspark/pytest it looks like any configs set here
     # can be reset in the middle of a test if specific operations are done (some types of cast etc)
     # enableHiveSupport() is needed for parquet bucket tests
-    _s = SparkSession.builder \
+    #TODO need to figure out a better way to do this optionally
+    import os
+    _sb = SparkSession.builder
+    _sb.config('spark.master', 'local') \
+            .config('spark.ui.showConsoleProgress', 'false') \
+            .config('spark.driver.extraClassPath', os.environ['EXTRA_CP']) \
+            .config('spark.sql.session.timeZone', 'UTC') \
+            .config('spark.sql.shuffle.partitions', '12') \
```

Comment:
Because this is based off of my hack I want to be sure that all of the new configs here are things that are set through an environment variable or through a config file.  This code needs to be generic enough that we can run this against really any spark cluster.  Things like `spark.driver.extraClassPath` I am setting here as a quick way to test that this works, but is not something that we would want to set if we were running on a real distributed cluster.